recipes/jupyterlite/LICENSE

```diff
@@ -0,0 +1,29 @@
+BSD 3-Clause License
```

Comment:
if kept in the same recipe as `jupyterlite-core`, can reference that one for now instead of vendoring a license.

Reply:
yes, and it should be fixed in `0.1.1`.